### PR TITLE
unconditionally add remote_file label to remote_files

### PR DESCRIPTION
**Pull request by nmpowell**
So we can query for all remote_files like so: `plz query alltargets --hidden --include=remote_file //...`

### CODE REVIEW

**Comment by nmpowell** [comment]
Other suggestions for how we could do this welcome!

File: rules/misc_rules.build_defs [hunk 1]
```diff
@@ -457,6 +457,7 @@ def remote_file(name:str, url:str|list, hashes:list=None, out:str=None, binary:b
     if bool(username) != bool(password_file):
         fail(f"Both or neither of username and password_file should be specified")
 
+    lables = labels + ["remote_file"]
```
Comment (peterebden) [inline rules/misc_rules.build_defs:460]: lables -> labels

The perils of dynamic languages!

Comment (peterebden) [reply rules/misc_rules.build_defs:460]: Also WYH: could you change this to a single statement please? i.e.
```
labels += [
    ...
]
```
mildly nicer than repeating the initial addition each time!

**Comment by peterebden** [comment]
Hmmmm also looks like we do have a `remote` label on them all here: https://github.com/thought-machine/please/blob/498599b3573e7c2422b3bde4be261aafbe2459f0/src/parse/asp/targets.go#L136

I'm not fussed about the specific name of the two, but would rather have just one. (Changing will need a little time though...)

**Comment by nmpowell** [comment]
I think we can do what we want with `plz query print //third_party/python3/... --include remote`

**Pull request closed without merging**